src/reducers/locations.js

```diff
@@ -40,6 +42,18 @@ function constructNewStateWithUpdatedPhones(state, action, newPhones, location,
   };
 }
 
+export const locationErrorsReducer = (state = {}, action) => {
```

Comment:
Btw, is it only locations that can break a page by failing to load, or are there cases where other errors actions or pieces of state could do the same?

Reply:
I need to look at this more closely. I would start by adding error handlers to other functions in `services/api.js`, adding logic to the reducer to ensure that they propagate the error into the state, and then looking at the components that use the state where a response is expected. I can create a separate task to track this.